dds_web/api/project.py

```diff
@@ -169,33 +170,27 @@ def post(self):
                 )
             elif new_status == "Archived":
                 is_aborted = json_input.get("is_aborted", False)
+                is_queue_operation = True
                 new_status_row, delete_message = self.archive_project(
-                    project=project, current_time=curr_date, aborted=is_aborted
+                    project=project,
+                    current_time=curr_date,
+                    aborted=is_aborted,
                 )
             else:
                 raise DDSArgumentError(message="Invalid status")
 
-            try:
-                project.project_statuses.append(new_status_row)
-                project.busy = False  # TODO: Use set_busy instead?
-                db.session.commit()
-                flask.current_app.logger.info(
-                    f"Busy status set. Project: '{project.public_id}', Busy: False"
+            return_message = ""
+
+            if not is_queue_operation:
+                # If operations was not handled by a queue, commit the changes
+                self.update_project_status(project=project, new_status_row=new_status_row)
+                return_message += f"Project {project.public_id} updated to status {new_status}"
+            else:
+                return_message += (
+                    f"Project {project.public_id} has started, in the background, the process of being updated to {new_status}"
+                    + (" (Aborted)" if is_aborted else "")
+                    + ". It may take some time to complete"
                 )
```

Comment:
I'd like to change the message to this, it makes it clearer. Also, the aborted part is removed here since aborted is always a substatus of archived and archived is always queued operation.
```suggestion
                return_message += (
                    f"The status of project {project.public_id} is being updated to {new_status}. The DDS is handling this in the background.  It may take some time to complete."
                )
```